python/ray/util/ray_executor.py

```diff
@@ -0,0 +1,277 @@
+import itertools
+import time
+from concurrent.futures import Executor
+
+import ray
+from ray.util.annotations import PublicAPI
+
+
+@PublicAPI
```

Comment:
Can you mark it as alpha version? @PublicAPI(stability="alpha")? We can bump the stability as time goes. Are you guys already using this in the production?

Reply:
No, we're not using it on production, I added the `alpha` stability tag